backend/tournesol/views/video.py

```diff
@@ -41,8 +41,11 @@ def list(self, request, *args, **kwargs):
         queryset = Video.objects.all()
         search = request.query_params.get('search')
         if search:
-            queryset = queryset.filter(Q(name__icontains=search) |
-                                       Q(description__icontains=search))
+            queryset = queryset.filter(
+                Q(name__search=search) | 
```

Comment:
Could you link to a doc that explains what this `__search` does ? It is probably useful to leave a comment here too.